Source/i_system.c

```diff
@@ -257,7 +257,7 @@ void I_Quit (void)
       G_CheckDemoStatus();
    M_SaveDefaults();
 
-#if defined(MY_SDL_VER)
+#if defined(MY_SDL_VER) && defined(_MSC_VER)

```

Comment:
SDL apps tend to compile as windowed under those tool chains, with stdout and stderr redirected to file, so in that case this tends to not apply anyway.